documentation/Using-the-SDK/Using-the-engine.md

```diff
@@ -0,0 +1,104 @@
+# Abstract
+
+This document outlines using the `Engine` to programmatically process data.
+
+# Overview
+
+The `Engine` is used to programmatically process data. There are three (3)
+major components to be aware of when using the `Engine`:
+- The `PluginSet`
+- The `DataSourceSet`
+- The `Engine` itself
+
+## Plugin Set
+
+The `PluginSet` is a collection of all of your plugins and extensions that can
+be used to process data from data sources (i.e. files.) You can create a new
```

Comment:
"i.e." -> "e.g."

Reply:
Thanks!

Reply:
Thanks, I tend to get the rules for parentheses and quotation marks confused.